
  20240812 now based on the java-21 branch, no longer on develop.


  Only here because patch file cannot be empty
--- a/poms/service/paisa-service-superpom/pom.xml
+++ b/poms/service/paisa-service-superpom/pom.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-    <modelVersion>4.0.0</modelVersion>
+    <modelVersion>4.0.0</modelVersion>
     <parent>
         <artifactId>paisa-superpom</artifactId>
         <groupId>ch.post.it.paisa</groupId>


  Mit mario debugging herausgefunden
  (index 5f8d03973..16e9554eb 100644)


--- a/alice/alice-docker/alice-docker-service-base/Dockerfile
+++ b/alice/alice-docker/alice-docker-service-base/Dockerfile
@@ -17,7 +17,7 @@ COPY install/jetty/             ${JETTY_HOME}/
 # copy slimmed libraries, generated in alice-docker-service-base pom
 COPY target/jetty/              ${JETTY_HOME}/
 
-RUN chmod 755 /usr/lib64/libmscan_api.so /usr/lib64/libmdis_api.so
+RUN chmod 755 /usr/lib/libmscan_api.so /usr/lib/libmdis_api.so
 RUN sed -i -e "s|platform_version|${PROJECT_VERSION}|g"  ${JETTY_HOME}/modules/AllowNullPathInfo.mod
 WORKDIR ${JETTY_HOME}
 
@@ -30,8 +30,13 @@ RUN apt-get update                                          && \
     apt-get clean all                                       && \
     # TODO: may be unused
     chmod +x ./bin/jetty.sh                                 && \
-    chmod +x ./bin/start-jetty.sh                                 && \
+    chmod +x ./bin/start-jetty.sh                           && \
-    chmod u+s /usr/bin/tcpdump
+    chmod u+s /usr/bin/tcpdump                              && \
+    # needed for tty devices (add to base-jetty)            && \
+    groupmod -g 18 dialout                                  && \
+    groupmod -g 63 audio                                    && \
+    # TODO remove as soon as it is fixed in base-jetty      && \
+    usermod -a -G 5,7,18,63,jetty,2001,2002 jetty
 
 # switch back to non-root
 USER jetty


diff --git a/alice/alice-docker/alice-docker-service-base/install/men_device_driver/usr/lib64/libmdis_api.so b/alice/alice-docker/alice-docker-service-base/install/men_device_driver/usr/lib/libmdis_api.so
similarity index 100%
rename from alice/alice-docker/alice-docker-service-base/install/men_device_driver/usr/lib64/libmdis_api.so
rename to alice/alice-docker/alice-docker-service-base/install/men_device_driver/usr/lib/libmdis_api.so


diff --git a/alice/alice-docker/alice-docker-service-base/install/men_device_driver/usr/lib64/libmscan_api.so b/alice/alice-docker/alice-docker-service-base/install/men_device_driver/usr/lib/libmscan_api.so
similarity index 100%
rename from alice/alice-docker/alice-docker-service-base/install/men_device_driver/usr/lib64/libmscan_api.so
rename to alice/alice-docker/alice-docker-service-base/install/men_device_driver/usr/lib/libmscan_api.so


